### DISCUSSION TITLE
Add a wherebelongstoMany method to align with other relation where methods

### DISCUSSION THREAD

**Discussion by fruitl00p** [laravel/framework#52852] [Ideas]
I was aligning our codebase with the various relation query usages and came across a "discrepency" (consistency is king in our codebase

When refactoring `->where('user_id', $user->id)` to `whereBelongsTo($user)` I came across a belongsToMany-relation and thought about the same reasoning: `->whereBelongsToMany($users)` but was stumped that this simply isnt there... 

Is there reasoning behind this? I'd love to do this.

**Comment by themegazord** [comment]
How's it going?

In Laravel, the whereBelongsTo method is used to simplify queries on "belongs to" relationships, while there is no whereBelongsToMany method because "belongs to many" relationships usually involve more complexity due to the pivot table.
